src/plusplus.coffee

```diff
@@ -142,7 +142,7 @@ module.exports = (robot) ->
     reasons = scoreKeeper.reasonsForUser(name)
 
     reasonString = if typeof reasons == 'object' && Object.keys(reasons).length > 0
-                     "#{name} has #{score} points. here are some raisins:" +
```

Comment:
It seems like a mistake a non-native speaker could make, since 'raisins' and 'reasons' have similar pronunciation. I do wonder why it has been in there for that long.